.gitignore

```diff
@@ -8,4 +8,6 @@ token-factory.did
 token.wasm
 /proptest-regressions
 
-/.DS_Store
\ No newline at end of file
+/.DS_Store
+
+Cargo.lock
```

Comment:
You added this to .gitignore, but the file is still commited. It should be either removed from git, or from .gitignore.

Reply:
Removed it from `.gitignore`.